src/services/ui/src/pages/welcome/index.tsx

```diff
@@ -121,6 +123,24 @@ export const Welcome = () => {
           </div>
         </div>
       </main>
+      <div className="bg-[#E1F3F8]"     >
+    <section className="  home-footer-container ">
+<div className="home-footer-angle-box  border-t-120 border-r-[100vw] border-transparent bottom-0 left-0 mb-20 h-117 border-solid border-t-24 border-r-screen"></div>
+<div className="home-footer-faq-callout flex justify-around items-center text-xl py-10 px-10 py-4 max-w-screen-xl mx-auto ">
+<div>
+  Do you have questions or need support?
+</div>
+<div className="ml-auto">
+<Button   variant='default'  onClick={()=>{
```

Comment:
on second thought, can we just use an a tag? something like:

<Button asChild><Link to="/faq">View FAQ</Link></Button>

Reply:
So you want us to go with a clickable text using an a tag, and the text should be "View FAQ" so then we will no longer need the button? Please let me know if i captured your thoughts correctly or incorrectly